modules/nf-core/cdhit/cdhit/main.nf

```diff
@@ -0,0 +1,36 @@
+process CDHIT_CDHIT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::cd-hit=4.8.1" : null)
```

Comment:
I'm not sure the template has been changed, but since #2654, Conda dependencies should be listed like this:
```suggestion
    conda "bioconda::cd-hit=4.8.1"
```